File: 240125/두 숫자의 차의 최솟값/minimum-difference-between-two-numbers.py
n = int(input())

arr = list(map(int, input().split()))

minminus = 99
for i in range(n-1):
    
    if minminus >= arr[i+1]- arr[i]:
        minminus = arr[i+1]- arr[i]
    
print(minminus)

File: 240923/두 숫자의 차의 최솟값/minimum-difference-between-two-numbers.cpp
#include <iostream>
using namespace std;

int main() {
    int n;
    cin >> n;
    int arr[n] = {};
    for (int i = 0; i < n; i ++) {
        cin >> arr[i];
    }

    int minminus = 100;
    for (int i = 0; i < n-1; i ++) {
        if (minminus >= arr[i+1] - arr[i]) {
            minminus = arr[i+1] - arr[i];
        }
    }
    cout << minminus;
    return 0;
}